tutorial/Lab2/README.md

```diff
@@ -16,7 +16,7 @@ In this part of the lab we will update the previously deployed application [Gues
 1. This is an **optional** step that is not technically required to update your running app. The reason for doing this step is "house keeping" - you want to have the correct files for the current configuration that you have deployed. This avoids making mistakes if you have future updates or even rollbacks. In this updated configuration, we remove the Redis slaves. To have the directory match the configuration, move/archive or simply remove the Redis slave files from the guestbook repo tree:
 
    ``` console
-   cd guestbook/v1
+   cd ~/guestbook/v1
```

Comment:
This might be risky if a user is not using the Cloud Shell for the environment or cloning into their home directory since it will pop up to the home directory. The previous assumption is that the user knew how to find the `guestbook` path.

Reply:
good catch, removed!